src/components/PetCard.tsx

```diff
@@ -42,24 +43,42 @@ export const PetCard = ({
     ) : null;
 
   return (
-    <Card className="w-[350px]">
-      <CardHeader>
-        <Avatar>
-          <AvatarImage src="https://github.com/shadcn.png" alt="@shadcn" />
-          <AvatarFallback>CN</AvatarFallback>
-        </Avatar>
-        <CardTitle className={`${font_accent.className} font-semibold text-3xl`}>{name}</CardTitle>
-        <p>{breed}</p>
-        <CardDescription>Born on {dateOfBirth}</CardDescription>
-        {genderDescription && (
-          <CardDescription className="flex items-center gap-1">{genderDescription}</CardDescription>
-        )}
-      </CardHeader>
-      <CardContent>
-        <Link href={`/pets/${name}`}>
-          <Button>See more</Button>
-        </Link>
-      </CardContent>
-    </Card>
+    <Link href={`/pets/${name}`}>
+      <Card className="w-[300px] h-[350px] relative">
+        <CardHeader>
+          <Avatar className="absolute top-0 left-0 z-0">
+            <AvatarImage
+              src="https://images.pexels.com/photos/3777622/pexels-photo-3777622.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+              // src="https://images.pexels.com/photos/159492/wildlife-photography-pet-photography-dog-dog-runs-159492.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+              alt="@shadcn"
+            />
+            <AvatarFallback>
```

Comment:
tak samo tu, zamiast <AvatarFallback> (z shadcn) może użyjmy [<Suspense>? ](https://react.dev/reference/react/Suspense)
ogólnie to spoko, fajnie wygląda taka łapka w tle zanim się załaduje :)